src/logger.cc

```diff
@@ -41,14 +41,24 @@ void Logger::Log(void *payload, TSLogType type, const char *message_str) {
 
   Local<Value> argv[3] = { name, params, type_name };
   TryCatch try_catch(Isolate::GetCurrent());
-  Nan::Call(fn, fn->CreationContext()->Global(), 3, argv);
+
+  #if (V8_MAJOR_VERSION > 9 || (V8_MAJOR_VERSION == 9 && V8_MINOR_VERION > 4))
+    Nan::Call(fn, fn->GetCreationContext().ToLocalChecked()->Global(), 3, argv);
+  #else
+    Nan::Call(fn, fn->CreationContext()->Global(), 3, argv);
+  #endif
```

Comment:
Just checking - does NAN not provide a wrapper for this API that works across V8 versions?

If not, could you extract a conditionally-defined `GetGlobal` function from this, which you'd use in all of the changed call sites?